app/routes/agent-details.js

```diff
@@ -133,18 +134,23 @@ module.exports = [
             mobileError,
             landlineError,
             address1Error,
+            address2Error,
```

Comment:
remove address2Error from here as we are not validating or handling any address2 error